config/secrets/sc-secrets.yaml

```diff
@@ -10,6 +10,10 @@ grafana:
   clientSecret: somelongsecret
   opsClientSecret: somelongsecret
 harbor:
+  # persistence:
+  #   swift:
+  #     username: "set-me"
+  #     password: "set-me"
```

Comment:
Sops doesn't like comments directly under an object key, see https://github.com/mozilla/sops/issues/384 I think we need to use sops 3.7 for this to work. (Maybe this is just a matter of telling people to use it and update the docs)

Please move it

Reply:
My bad! I ran with 3.7 locally. :sweat_smile:  Added the comment at the bottom.